views/Account.react.js

```diff
@@ -112,8 +113,8 @@ class AccountPage extends Component {
     })
   }
   editUserInfo = () => {
-    let uid = this.state.uid
-    let user = {}
+    const { uid } = this.state
+    const user = {}
```

Comment:
you can combine this with the lines below

```suggestion
    const user = {
      uid,
      display_name: this.state.display_name,
      title: this.state.title,
      department: this.state.department,
      company: this.state.company,
      mail: this.state.mail,
      icon: this.state.icon,
    }
```

Reply:
This is outside of the scope of this ticket